Visual Studio/Realm.vcxproj

```diff
@@ -2599,4 +2599,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
+</Project>
```

Comment:
Just to be clear, it is not a warning. It is just a way to show that there is no new-line on the last line of this file. It would difficult to otherwise see when rendering text inside a box. 😃